prefect_aws/waiters/batch.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
I think it be easier to make this a `.py` file with a dictionary. It simplifies the packaging and loading of this information.

Reply:
Are these definitions even necessary, and do they still apply in Prefect 2, or just valid in Prefect 1? I'm unfamiliar how this gets used by users.

Reply:
It appears that they aren't needed anymore. It looks like they were used to wait for AWS Batch jobs, but the Batch boto3 client offers a built in waiter.